src/Formatter/TomlFormatter.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Crell\Serde\Formatter;
+
+use Crell\Serde\Attributes\ClassSettings;
+use Crell\Serde\Attributes\Field;
+use Crell\Serde\Deserializer;
+use Devium\Toml\Toml;
+use Devium\Toml\TomlError;
+
+class TomlFormatter implements Formatter, Deformatter, SupportsCollecting
+{
+    use ArrayBasedFormatter;
+    use ArrayBasedDeformatter;
+
+    /**
+     * Constructor parameters map directly to the devium/toml component's encode() and decode() methods.
+     *
+     * @see Toml::encode()
+     * @see Toml::decode()
+     */
```

Comment:
Well, this docblock isn't true. :smile:   I assume copy-pasta from YAML?

Reply:
Of course copy-paste :)

Reply:
It's still wrong.  There's no constructor parameters, so they cannot map to anything.

And if the constructor is fully empty, just omit it.

Reply:
And now?